packages/openapi-ts/bin/index.cjs

```diff
@@ -113,6 +118,8 @@ async function start() {
       userConfig.logs.level = 'silent';
     }
 
+    userConfig.logs.file = userConfig.logFile;
```

Comment:
Which values will this possibly have? Won't there be some case where it's setting an undefined or invalid value? Thinking if it should be guarded

Reply:
![image](https://github.com/user-attachments/assets/8f672d65-2d29-4c67-b5a5-cc314aee91a9)

The tooling seems to automatically toggle it true/false as a boolean due to it's form, so it should always be a boolean